jobserver/authorization/admins.py

```diff
@@ -19,17 +20,25 @@ def get_admins():
     return [u.strip() for u in admin_users if u]
 
 
+@transaction.atomic()
 def ensure_admins(usernames):
     """
     Given an iterable of username strings, ensure they have the SuperUser role
     """
-    admins = User.objects.filter(username__in=usernames)
+    for user in User.objects.all():
+        try:
+            user.roles.remove(SuperUser)
+        except ValueError:
```

Comment:
Why would this raise `ValueError`?

Reply:
`roles` is a list and calling `remove(thing)` when `thing` isn't in the list raises that.  The idea with this was to make sure no users have `SuperUser` before adding it to the Users we want.